docs/DeveloperGuide.md

```diff
@@ -461,32 +566,59 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
   Use case ends.
 
-* 3a. The given student is invalid/cannot be found.
+* 3a. The inputted student's name is invalid or cannot be found.
 
     * 3a1. EduManage shows an error message.
 
-      Use case resumes at step 2.
+      Use case resumes at step 3.
+
+***
+
+**Use case: Add a task for a student**
 
-* 3b. Multiple school level subjects were given.
+**MSS**
+
+1.  Tuition teacher requests to list students.
+2.  EduManage shows a list of students.
+3.  Tuition teacher requests to add a task to a specific student in the list by inputting the student's name, task description and task deadline.
```

Comment:
Tuition teacher requests to add a task to a specific student's task list by inputting the student's name, task description and task deadline.
 
 Make it a bit clearer that task is added to the task list of the student